### DISCUSSION TITLE
حل مشکل خطای 403 گوگل بدون استفاده از وارپ یا غیرفعال کردن ipv6

### DISCUSSION THREAD

**Discussion by parhawm** [Gozargah/Marzban#734] [Show and tell]
سلام دوستان. همونطور که میدونید گوگل با آیپی ورژن ۶ دیتاسنتر هتزنر خطا میده و باز نمیشه. برای اینکه این مشکل رو حل کنید فقط کافیه اولویت استفاده از ipv6 رو کم کنید و نیازی به استفاده از وارپ نیست.
این رو توی اوت باند ها بزارید(اگر فریدوم دارید تنظیماتی که توشه رو اضافه کنید):   

```
    {
      "protocol": "freedom",
      "tag": "DIRECT",
      "settings": {
        "domainStrategy": "UseIPv4"
      }
    }
```

این هم باید بخشی از تنظیمات اصلی باشه:

```
  "dns": {
    "servers": ["1.1.1.1", "8.8.8.8"], // میتونید به دلخواه سروری که میخواید رو بزارید به عنوان مثال آیپی سرور دی ان اس هتزنر
    "queryStrategy": "UseIPv4",
    "tag": "dns_inbound"
  }
```

به این صورت xray ترجیح میده از ipv4 استفاده کنه. اگر سایتی ipv4 نداشته باشه هم به درستی کار میکنه و باز میشه.
**حتما حتما باید هر دوتا مخصوصا قسمت دوم به درستی باشه تا کار کنه**
دقت کنید تو حالت عادی xray دامنه ها رو از سیستم میخونه و سیستم به طور پیشفرض ipv6 رو ترجیح میده. در اون صورت سیستم تعیین کننده اولویت ipv6 هست.
اینجا از سرور dns درون خودش استفاده میکنه و اولویت بر اساس کانفیگ قابل تنظیمه.

**Comment by MirzaT1992** [comment]
به این ترتیب بزنید با تشکر از دوستمون که وقت گذاشت
![image](https://github.com/Gozargah/Marzban/assets/142059850/8e3bd3f2-b13e-4ec6-8474-b5bc77388879)

![image](https://github.com/Gozargah/Marzban/assets/142059850/50d04080-67e5-43d9-b040-e1805ad1438d)

**Comment by PouriArc** [comment]
بی نظیر بود آقا , دس خوش

**Comment by BehnamAB** [comment]
عالی بود

**Comment by Meytiz** [comment]
سلام من این کار رو انجام دادم اما باز سایت https://analytics.google.com/analytics/web/ باز نمیشه ارور 403 میده راه حلی دارید کانفیگ CDN

**Comment by Mjzaret** [comment]
من فقط از این قسمت کد استفاده کردم و مشکلم حل شد.
 {
      "protocol": "freedom",
      "tag": "DIRECT",
      "settings": {
        "domainStrategy": "UseIPv4"
      }
    }

آیا قسمت دوم که مربوط به DNS هست واجبه انجام بشه یا نیازی نیست؟
و اینکه چطور میشه برای این قسمت تنظیمات بیشترش رو پیدا کرد؟
منظورم اینه هر بخش از کدهای جیسون Xray شامل چه گزینه هایی هست؟

**Comment by karimzzzm** [comment]
عالیه دستت درد نکنه

**Comment by ZarinNegah** [comment]
سلام
من دقیقا انجام دادم و مشکلم حل نشد
و همچنان سایت زیر خطا 403 میده
ads.google.com

**Reply by ImMohammad20000** [reply]
اون بخاطر تحریم گوگله

**Comment by waxilu** [comment]
```
"dns": {
    "servers": ["1.1.1.1", "8.8.8.8"], 
    "queryStrategy": "UseIPv4",
    "tag": "dns_inbound"
  }
```
  
  
  این قسمت نزاریم چ اتفاقی میفته؟ چون برای من بدون این بخش داره درست کار میکنه ممکنه تو اپ خاصی به مشکل بخوره؟

**Comment by am-fallah** [comment]
سلام
ایا این تنظیمات روی پنل S-UI و پروتکل هیستریا ۲ هم قابل اعماله؟

**Comment by bombcreator** [comment]
سلام
این راه حل تا مدتی کار می کرد اما الان همچنان خطای 403 رو می گیرم. راه دیگه ای وجود دارد؟

**Reply by MirzaT1992** [reply]
مشکلی نداره هنوز

**Comment by Eslender73** [comment]
یه مشکلی که هست باید sniffing در حداقل یک سمت روشن باشه تا کار کنه

**Comment by payammasroori-crypto** [comment]
> سلام دوستان. همونطور که میدونید گوگل با آیپی ورژن ۶ دیتاسنتر هتزنر خطا میده و باز نمیشه. برای اینکه این مشکل رو حل کنید فقط کافیه اولویت استفاده از ipv6 رو کم کنید و نیازی به استفاده از وارپ نیست.
> این رو توی اوت باند ها بزارید(اگر فریدوم دارید تنظیماتی که توشه رو اضافه کنید):   
> 
> ```
>     {
>       "protocol": "freedom",
>       "tag": "DIRECT",
>       "settings": {
>         "domainStrategy": "UseIPv4"
>       }
>     }
> ```
> 
> این هم باید بخشی از تنظیمات اصلی باشه:
> 
> ```
>   "dns": {
>     "servers": ["1.1.1.1", "8.8.8.8"], // میتونید به دلخواه سروری که میخواید رو بزارید به عنوان مثال آیپی سرور دی ان اس هتزنر
>     "queryStrategy": "UseIPv4",
>     "tag": "dns_inbound"
>   }
> ```
> 
> به این صورت xray ترجیح میده از ipv4 استفاده کنه. اگر سایتی ipv4 نداشته باشه هم به درستی کار میکنه و باز میشه.
> **حتما حتما باید هر دوتا مخصوصا قسمت دوم به درستی باشه تا کار کنه**
> دقت کنید تو حالت عادی xray دامنه ها رو از سیستم میخونه و سیستم به طور پیشفرض ipv6 رو ترجیح میده. در اون صورت سیستم تعیین کننده اولویت ipv6 هست.
> اینجا از سرور dns درون خودش استفاده میکنه و اولویت بر اساس کانفیگ قابل تنظیمه.

- [ ] [](

**Comment by payammasroori-crypto** [comment]
سلام دوستان. همونطور که میدونید گوگل با آیپی ورژن ۶ دیتاسنتر هتزنر خطا میده و باز نمیشه. برای اینکه این مشکل رو حل کنید فقط کافیه اولویت استفاده از ipv6 رو کم کنید و نیازی به استفاده از وارپ نیست.
این رو توی اوت باند ها بزارید(اگر فریدوم دارید تنظیماتی که توشه رو اضافه کنید):

    {
      "protocol": "freedom",
      "tag": "DIRECT",
      "settings": {
        "domainStrategy": "UseIPv4"
      }
    }
این هم باید بخشی از تنظیمات اصلی باشه:

  "dns": {
    "servers": ["1.1.1.1", "8.8.8.8"], // میتونید به دلخواه سروری که میخواید رو بزارید به عنوان مثال آیپی سرور دی ان اس هتزنر
    "queryStrategy": "UseIPv4",
    "tag": "dns_inbound"
  }
به این صورت xray ترجیح میده از ipv4 استفاده کنه. اگر سایتی ipv4 نداشته باشه هم به درستی کار میکنه و باز میشه.
حتما حتما باید هر دوتا مخصوصا قسمت دوم به درستی باشه تا کار کنه
دقت کنید تو حالت عادی xray دامنه ها رو از سیستم میخونه و سیستم به طور پیشفرض ipv6 رو ترجیح میده. در اون صورت سیستم تعیین کننده اولویت ipv6 هست.
اینجا از سرور dns درون خودش استفاده میکنه و اولویت بر اساس کانفیگ قابل تنظیمه.

**Comment by payammasroori-crypto** [comment]
سلام دوستان. همونطور که میدونید گوگل با آیپی ورژن ۶ دیتاسنتر هتزنر خطا میده و باز نمیشه. برای اینکه این مشکل رو حل کنید فقط کافیه اولویت استفاده از ipv6 رو کم کنید و نیازی به استفاده از وارپ نیست.
این رو توی اوت باند ها بزارید(اگر فریدوم دارید تنظیماتی که توشه رو اضافه کنید):

    {
      "protocol": "freedom",
      "tag": "DIRECT",
      "settings": {
        "domainStrategy": "UseIPv4"
      }
    }
این هم باید بخشی از تنظیمات اصلی باشه:

  "dns": {
    "servers": ["1.1.1.1", "8.8.8.8"], // میتونید به دلخواه سروری که میخواید رو بزارید به عنوان مثال آیپی سرور دی ان اس هتزنر
    "queryStrategy": "UseIPv4",
    "tag": "dns_inbound"
  }
به این صورت xray ترجیح میده از ipv4 استفاده کنه. اگر سایتی ipv4 نداشته باشه هم به درستی کار میکنه و باز میشه.
حتما حتما باید هر دوتا مخصوصا قسمت دوم به درستی باشه تا کار کنه
دقت کنید تو حالت عادی xray دامنه ها رو از سیستم میخونه و سیستم به طور پیشفرض ipv6 رو ترجیح میده. در اون صورت سیستم تعیین کننده اولویت ipv6 هست.
اینجا از سرور dns درون خودش استفاده میکنه و اولویت بر اساس کانفیگ قابل تنظیمه.
